googleapiclient/http.py

```diff
@@ -24,34 +24,35 @@
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
-import StringIO
+from six import BytesIO, StringIO
+from six.moves.urllib.parse import urlparse, urlunparse, quote, unquote
```

Comment:
I tried looking for this in Google's style guide, but have been unable to find it anymore.  I remember reading it before, and I recall, in general `Follow PEP8, except use 2 spaces instead of 4`, but that's it.

Do you know where to find this?  We should probably add a link to it in the README.

Maybe @craigcitro or @nathanielmanistaatgoogle can help out?